pallets/capacity/src/types.rs

```diff
@@ -15,7 +15,7 @@ use sp_runtime::{
 use sp_std::vec::Vec;
 
 /// How much, as a percentage of staked token, to boost a targeted Provider when staking.
-/// this value should be between [0,100]
+/// this value should be between 0,100 inclusive
```

Comment:
the linter didn't like the "fake link" that went nowhere :/

Reply:
That's the rust docs for you. They make it easy to link things. For future reference placing back ticks around it should mark it as code instead of as a link